samples/Mvc.ControllerForMultipleAreasSample/MultipleAreasAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace ControllerForMultipleAreasSample
+{
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = false, Inherited = true)]
+    public class MultipleAreasAttribute : Attribute
+    {
+        public MultipleAreasAttribute(string[] areaNames)
```

Comment:
a handy pattern for these things is to do something like `MulipleAreasAttribute(string a1, string a2, params string[] a3...)` that makes it clear you're supposed to use at least 2 values.